hathor/transaction/transaction.py

```diff
@@ -40,10 +41,19 @@
 _SIGHASH_ALL_FORMAT_STRING = '!BBBBB'
 
 
+# used when (de)serializing token information
+class TokenInfoVersion(IntEnum):
+    DEPOSIT = 1
+    FEE = 2
+
+
 class TokenInfo(NamedTuple):
     amount: int
     can_mint: bool
     can_melt: bool
+    version: TokenInfoVersion | None
+    spent_outputs: list[TxOutput]
+    outputs: list[TxOutput]
```

Comment:
This point about the `TokenInfo` is to keep information about the balances and token authorities. Why are you adding a copy of the outputs to this data structure?